package.json

```diff
@@ -66,6 +66,7 @@
     "regenerator-runtime": "^0.12.1"
   },
   "dependencies": {
+    "axios": "^0.19.2",
```

Comment:
Please use `request` to implement API requests instead as we don't need to use a third-party library. 

Refer to some of the code in `src/api` directory